### ISSUE TITLE
Slider navigation appears even if not needed, in specific situation

### ISSUE THREAD

**Issue by razvanioan** [uikit/uikit#3490]
### UIkit version

3.0.0-rc.10

### Browser
- Firefox latest (Mac latest)
- Safari latest (Mac latest)

### Reproduction Link
https://codepen.io/qdev/pen/MBePpo (JS code makes sure the slider container width will not be divisible by 4)

### Steps to reproduce
- if slider configured to display `x` number of items per set (grid width on specific size)
- if slider has exactly `x` items (navigation shouldn't appear)
- if slider container width doesn't divide exactly with `x`
- navigation appears although not needed
- if window resized in order to have a slider container width divisible by `x`, navigation disappears

### What is Expected?
- don't show slider navigation (next / prev arrows, dotnav)

### What is actually happening?
- slider navigation present even if all the items are visible (no slide needed)

### Possible cause
- maxItems calculated wrongly maybe due to rounded up value of `getWidth(this.list)` (for a 1663 pixels window width it returns 1664)

**Comment by razvanioan** [comment]
Nothing on this yet ?

Here's another demo, directly on UIkit Slider documentation page

- Browse to UIkit Slider documentation page, and scroll to [Navigation section](https://getuikit.com/docs/slider#navigation), `dotnav` example
- be sure to have a browser window width bigger than `@m`, in order to show 4 slides per set
- delete all last `li` items, and keep only 4 of them (only one set visible) or execute the code bellow in console to automatically do this
- the navigation (arrows & dots) will still be visible although not needed

```javascript
document.querySelectorAll('.uk-slider')[4].querySelectorAll('li').forEach((li, idx) => {if (idx > 3) li.parentNode.removeChild(li)})
```

**Comment by janschoenherr** [comment]
Sorry, cannot reproduce the issue. In both of your examples the navigation is hidden for me.

**Comment by razvanioan** [comment]
What browser do you use ? I just found that on Chrome it works as expected, but on Firefox & Safari not (I will update browser section on initial post)

Here the screenshots (Firefox, Safari & Chrome)

![qdev 2018-08-09 12 20 59](https://user-images.githubusercontent.com/2914226/43890556-c7e9b96c-9bcf-11e8-822a-3639b33e698e.png)
![qdev 2018-08-09 12 24 28](https://user-images.githubusercontent.com/2914226/43890557-c826412a-9bcf-11e8-890c-1443139d1d1f.png)
![qdev 2018-08-09 12 26 42](https://user-images.githubusercontent.com/2914226/43890558-c8443a40-9bcf-11e8-92d9-f94c38c55b53.png)

**Comment by janschoenherr** [comment]
Thanks, I can reproduce the issue in FF.

**Comment by janschoenherr** [comment]
Fixed with 185749af9fb7e731ba66bec8676b9bda672d35d8
